packages/trzsz-ssh/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://trzsz.github.io/ssh
+TERMUX_PKG_DESCRIPTION="An openssh client alternative"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.1.21"
+TERMUX_PKG_SRCURL=https://github.com/trzsz/trzsz-ssh/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=598d749e50af298700bfaa416690383bb16b9fc3d15ec66951857323726e2124
+TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
You can add `trzsz-go` as a recommended package for `trzsz-ssh`.
```suggestion
TERMUX_PKG_RECOMMENDS='trzsz-go'
TERMUX_PKG_AUTO_UPDATE=true
```

And likewise in `packages/trzsz-go/build.sh`
```bash
TERMUX_PKG_REVISION=1
TERMUX_PKG_RECOMMENDS='trzsz-ssh'
```

Reply:
Eh quick fix, let me push a commit revision quick.